examples/getting_started/user_config.yaml.example

```diff
@@ -34,17 +60,39 @@ SIMULATION_CONTROLLER:
     SPHERE_ACTOR_NAME: "SphereActor"
     CAMERA_ACTOR_NAME: "CameraActor"
     ACTION_COMPONENTS: ["add_force", "add_rotation"] # "add_force", "add_rotation"
-    OBSERVATION_COMPONENTS: ["camera", "position", "rotation"] # "camera", "position", "rotation"
+    OBSERVATION_COMPONENTS: ["location", "rotation", "camera"] # "location", "rotation", "camera"
     STEP_INFO_COMPONENTS: ["debug"] # "debug"
-    SPAWN_POSITION_X: 0.0
-    SPAWN_POSITION_Y: 0.0
-    SPAWN_POSITION_Z: 100.0
+    SPAWN_LOCATION_X: 0.0
+    SPAWN_LOCATION_Y: 0.0
+    SPAWN_LOCATION_Z: 20.0
     CAMERA:
       RENDER_PASSES: ["final_color"] # "depth", "final_color", "normal", "segmentation"
       IMAGE_HEIGHT: 512
       IMAGE_WIDTH: 512
       FOV: 90.0
 
+  POINT_GOAL_NAV_TASK:
```

Comment:
I don't think you need this because we're not using this task in this example any more.

Reply:
Yup, removed it